src/Features/CSharp/Portable/Snippets/CSharpConvertToLSPSnippetService.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Text;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Snippets;
+
+namespace Microsoft.CodeAnalysis.CSharp.Snippets
+{
+    [ExportLanguageService(typeof(ConvertToLSPSnippetService), LanguageNames.CSharp), Shared]
```

Comment:
i'm curious why this is a language service.  what is language specific about conversion?

Reply:
removed